lldb/test/API/functionalities/data-formatter/data-formatter-stl/generic/vector/main.cpp

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+#include <string>
+#include <vector>
+typedef std::vector<int> int_vect;
+typedef std::vector<std::string> string_vect;
+
+template <class T>
+void by_ref_and_ptr(std::vector<T> &ref, std::vector<T> *ptr) {
+  // Stop here to check by ref
+  return;
+}
+
+int main() {
+  int_vect numbers;
+  (numbers.push_back(1));  // break here
```

Comment:
I'm not sure what are the parens for, but I wouldn't want my name to go down in history with them :P